docs/application/tizen-studio/release-notes/2-5-release-notes.md

```diff
@@ -0,0 +1,82 @@
+# Tizen Studio 2.5 Release Notes
+
+
+## IDE and Tools
+
+
+### New Features
+
+-   SDB
+	-  wait-for-device command added, when this command is issued sdb waits indefinitely for the target to come to 'device' state.
+-   RT IDE Certificate Manager
+	-   Integrated Certificate Manager into RT IDE
+-   Baseline SDK Size Optimizations
+	-   Optimized Baseline SDK size from 480MB to 320MB
+
+### Fixed Bugs
+-   Certificate Manager
+	-   Added validation of .pem & .der files while importing as a new IoT profile in Certificate Manager
+
+-   Installer  and Package Manager
+	-   Fix extension download failure when url starts with https.
+	-   Fixed a issue of uninstaller window getting closed while uninstallation.
+	-   Fixed a issue of target directory not being able to set after invalid path.
+-   Base IDE
+	-   Supporting "Show in History" view of eclipse team service. 
+-   CLI
+	-   Fixed CLI Build failure for IoT Headless App.
+	> **Note**
```

Comment:
Note tag style:
> **Note**
>
> If you have projects with old CLI build scripts, you must re-create the build scripts by running *export to CLI* before running the CLI commands.

It is recommended to use *export to CLI* if the purpose is to emphasize.
I changed Export to export, please verify.

Reply:
Hello, @Reni-90 
As in Markdown document it is hard to recognize the italic font at once, how about using `code tag style` or **bold font**?

Reply:
If it is a command then it is better to use `code tag style`, or bold font is also fine.